modin/pandas/base.py

```diff
@@ -2466,7 +2466,7 @@ def reorder_levels(self, order, axis=0):  # noqa: PR01, RT01, D200
         Rearrange index levels using input order.
         """
         axis = self._get_axis_number(axis)
-        new_labels = self.axes[axis].reorder_levels(order)
+        new_labels = self._query_compiler.get_axis(axis).reorder_levels(order)
```

Comment:
maybe worth adding a private method `._get_axis()` for shortness?

Reply:
Will add